airflow/contrib/operators/bigquery_operator.py

```diff
@@ -121,3 +124,111 @@ def on_kill(self):
         if self.bq_cursor is not None:
             self.log.info('Canceling running query due to execution timeout')
             self.bq_cursor.cancel_query()
+
+
+class BigQueryCreateEmptyTableOperator(BaseOperator):
+    """
+    Creates a new, empty table in the specified BigQuery dataset,
+    optionally with schema.
+
+    The schema to be used for the BigQuery table may be specified in one of
+    two ways. You may either directly pass the schema fields in, or you may
+    point the operator to a Google cloud storage object name. The object in
+    Google cloud storage must be a JSON file with the schema fields in it.
+    You can also create a table without schema.
+
+    :param project_id: The project to create the table into.
+    :type project_id: string
+    :param dataset_id: The dataset to create the table into.
+    :type dataset_id: string
+    :param table_id: The Name of the table to be created.
+    :type table_id: string
+    :param schema_fields: If set, the schema field list as defined here:
+        https://cloud.google.com/bigquery/docs/reference/v2/jobs#configuration.load
+    :type schema_fields: list
+    :param schema_object: If set, a GCS object path pointing to a .json file that
+        contains the schema for the table.
+    :type schema_object: string
```

Comment:
do you want to merge schema_object and schema_object_gcs_bucket as gcs_schema_object?